app/assets/stylesheets/sephora_style_guide/app/_modals.scss

```diff
@@ -0,0 +1,18 @@
+#modals {
+  section {
+    display: inline-block;
+    width: 50%;
+  }
+  pre {
+    white-space: pre-wrap;
+    white-space: -moz-pre-wrap;
```

Comment:
Vendor prefixes are not needed since we use `autoprefixer` which includes them by default.